Directory.Build.props

```diff
@@ -9,6 +9,8 @@
     <_DefaultTargetFrameworks>net8.0-android</_DefaultTargetFrameworks>
     <_DefaultNetTargetFrameworks>net8.0</_DefaultNetTargetFrameworks>
 
+    <CheckEolWorkloads>false</CheckEolWorkloads>
+
```

Comment:
Do you need this? Do you get a warning/error if you remove it?

Reply:
I was getting warnings for `net8.0` when built with `net10.0`